node/config.go

```diff
@@ -119,6 +120,8 @@ type Config struct {
 	// exposed.
 	HTTPModules []string `toml:",omitempty"`
 
+	HTTPTimeouts rpc.HTTPTimeouts
```

Comment:
Please properly document this field. All other fields are extensively detailed, let's not ruing the streak ;)